src/makeRobotAccountant.js

```diff
@@ -6,7 +6,16 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  return (first) => {
+    count++;
+
+    return (second) =>
+      (count >= 3 && count % 2 === 0)
+        ? 'Bzzz... Error!'
+        : first + second;
```

Comment:
Try not to combine && and ternary operators.
```suggestion
    return (second) => {
      const shouldReturnError = count >= 3 && count % 2;
      
      return shouldReturnError
        ? 'Bzzz... Error!'
        : first + second;
    }
```